test/modules/TestResultCoordinator/TestReportUtil.cs

```diff
@@ -154,9 +154,21 @@ internal static async Task UploadLogsAsync(string iotHubConnectionString, Uri bl
             logger.LogInformation("Send upload logs request to edgeAgent.");
 
             ServiceClient serviceClient = ServiceClient.CreateFromConnectionString(iotHubConnectionString);
+
             CloudToDeviceMethod uploadLogRequest =
-                new CloudToDeviceMethod("UploadModuleLogs")
-                .SetPayloadJson($"{{ \"schemaVersion\": \"1.0\", \"sasUrl\": \"{blobContainerWriteUri.AbsoluteUri}\", \"items\": [{{ \"id\": \".*\", \"filter\": {{}} }}], \"encoding\": \"gzip\", \"contentType\": \"json\" }}");
+                new CloudToDeviceMethod("UploadModuleLogs");
+            uploadLogRequest = logUploadDuration.Match<CloudToDeviceMethod>(
+            (TimeSpan logUploadDuration) =>
+            {
+                DateTime logStart = DateTime.UtcNow - logUploadDuration;
+                uploadLogRequest = uploadLogRequest.SetPayloadJson($"{{ \"schemaVersion\": \"1.0\", \"sasUrl\": \"{blobContainerWriteUri.AbsoluteUri}\", \"items\": [{{ \"id\": \".*\", \"filter\": {{\"since\": \"{logStart.ToString("O")}\"}} }}], \"encoding\": \"gzip\", \"contentType\": \"json\" }}");
```

Comment:
You could just return from line 164, but this is a pretty minor stylistic change.

Reply:
Same for line 169.